core/pom.xml

```diff
@@ -34,12 +34,12 @@
     <dependency>
       <groupId>org.graalvm.js</groupId>
       <artifactId>js</artifactId>
-      <version>22.0.0</version>
+      <version>22.2.0</version>
```

Comment:
1) `22.3.0` is already available
2) what about introducing a property (i.e. `graalvm-js.version`) and reuse it twice?